free-programming-books-id.md

```diff
@@ -72,7 +72,7 @@
 
 ### MySQL
 
-* [Mysql For Your Application: Mysql Untuk Pemula](https://play.google.com/store/books/details/Onesinus_Saut_Parulian_3_Days_With_Mysql_For_Your?id=MbdTDwAAQBAJ) - Onesinus Saut Parulian, Google Play Books
+* [MySQL : Berbahasa Indonesia Pemula](https://play.google.com/store/books/details/Onesinus_Saut_Parulian_3_Days_With_Mysql_For_Your?id=MbdTDwAAQBAJ) - Onesinus Saut Parulian, Google Play Books
 * [Tutorial MySQL untuk Pemula Hingga Mahir](https://umardanny.com/tutorial-mysql-untuk-pemula-hingga-mahir-ebook-download-pdf/)
 
 ### Node.js
```

Comment:
Need 2 blank lines here as well as between "Python" and "Vue" section.

Also, there is another issue at hand that needs to be addressed #4606